src/SymfonyContainerAdapter.php

```diff
@@ -33,13 +30,8 @@ public function get($id)
         if ($this->container->hasParameter($id)) {
             return $this->container->getParameter($id);
         }
-        try {
```

Comment:
Symfony container is PSR-11 compliant so no need to adapt exceptions.

Reply:
True!